ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/NodeCvesOverviewPage.tsx

```diff
@@ -177,6 +186,30 @@ function NodeCvesOverviewPage() {
                     </FlexItem>
                 </Flex>
             </PageSection>
+            {scannerV4NodeScanResultsPossible && (
+                <PageSection variant="light" className="pf-v5-u-pt-0">
+                    <Alert
+                        isInline
+                        variant="info"
+                        title="Results may include Node CVEs obtained from Scanner V4"
+                        component="p"
+                    >
+                        <ExternalLink>
+                            <a
+                                href={getVersionedDocs(
+                                    version,
+                                    'operating/manage-vulnerabilities/scan-rhcos-node-host#understanding-node-cves-scanner-v4_scan-rhcos-node-host'
```

Comment:
The PR that would add this link is here:

@pedrottimark, shall I still shorten the link as suggested in this comment?

> Can you please replace with '/operating/managing-user-access#configure-short-lived-access' because I am in process of updating links for docs dot redhat dot com.

Reply:
@vikin91 The link on line 201 looks correct. I opened the Netlify preview and deleted `.html` to verify that a link in this form opens the doc at the heading.

Reply:
For your info I will merge change from openshift to redhat docs later today in #14178

Reply:
I'll set this for auto-merge (due to personal deatline), but I will check tomorrow whether the link needs any further adjustments.